src/Settings.php

```diff
@@ -444,6 +445,33 @@ protected function createLanguageSection($settings_key)
         }
         /** END LANGUAGE SECTION */
 
+        /** STYLING SECTION */
+        protected function createStylingSection($settings_key)
+        {
+            global $ADMWPP_LANG;
+
+            add_settings_section(
+                'admwpp_styling_settings',
+                "<span class='admwpp-section-title'>" . __('Styling', ADMWPP_TEXT_DOMAIN) . "</span>",
+                array($this, 'stylingSettingsSection'),
+                "admwpp_" . $settings_key . "_settings"
+            );
+
+            add_settings_field(
+                'admwpp_styles_settings',
+                __('Styles', ADMWPP_TEXT_DOMAIN) . ':',
+                array($this, 'settingsFieldInputBoolean'),
+                "admwpp_" . $settings_key . "_settings",
+                'admwpp_styling_settings',
+                array(
+                    'field'        => 'styles',
+                    'settings_key' => $settings_key,
+                    'info'         => 'Selecting this checkbox will disable selectric JS library and plugin styles.',
```

Comment:
can we specify the selectric library version please. you ca add it on a seperate line using `<br>`